File: Semester 1/Class07_Computer_Contest_Lv_2_Notes_and_Homework_Oct_23-24/lvl2_hw_sol/lvl2_pumpkin_patches.cpp
#include <bits/stdc++.h>
using namespace std;
int N, minx= 1e8, maxx=-minx, miny=1e8, maxy=-miny;
int main(){
    scanf("%d", &N);
    for(int i=0, x, y; i<N; i++){
        scanf("%d %d", &x, &y);
        minx = min(minx, x); maxx = max(maxx, x);
        miny = min(miny, y); maxy = max(maxy, y);
    }
    printf("%d\n", 2*(maxx-minx + maxy-miny));
}

File: Semester 1/Class07_Computer_Contest_Lv_2_Notes_and_Homework_Oct_23-24/lvl2_hw_sol/lvl2_pumpkin_patches.java
import java.util.*;
public class test2 {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int n = in.nextInt(), minX = Integer.MAX_VALUE, maxX = Integer.MIN_VALUE;
		int minY = Integer.MAX_VALUE, maxY = Integer.MIN_VALUE;
		for(int i=1; i<=n; i++) {
			int x = in.nextInt(), y = in.nextInt();
			minX = Math.min(minX, x);  maxX = Math.max(maxX, x);
			minY = Math.min(minY, y);  maxY = Math.max(maxY, y);
		}
		System.out.println(2*(maxX - minX) + 2*(maxY - minY));
	}
}